doc/modules/model_evaluation.rst

```diff
@@ -575,13 +585,18 @@ or *informedness*.
 
     * Our definition: [Mosley2013]_, [Kelleher2015]_ and [Guyon2015]_, where
       [Guyon2015]_ adopt the adjusted version to ensure that random predictions
-      have a score of :math:`0` and perfect predictions have a score of :math:`1`..
+      have a score of :math:`0` and perfect predictions have a score of :math:`1`.
     * Class balanced accuracy as described in [Mosley2013]_: the minimum between the precision
       and the recall for each class is computed. Those values are then averaged over the total
       number of classes to get the balanced accuracy.
     * Balanced Accuracy as described in [Urbanowicz2015]_: the average of sensitivity and specificity
       is computed for each class and then averaged over total number of classes.
 
+    There are some cases, where balanced accuracy does not avoid inflated performance
+    scores in imbalanced datasets (for example, when minority classes are detected well, but
+    majority classes aren't). In these cases, accuracy can be low, but balanced accuracy can
+    be high. Therefore [Opitz2024]_ recommend inspecting balanced accuracy together with accuracy.
```

Comment:
What do you think of the following reference?

[https://imbalanced-learn.org/stable/auto_examples/applications/plot_impact_imbalanced_classes.html](https://imbalanced-learn.org/stable/auto_examples/applications/plot_impact_imbalanced_classes.html)